scripts/i18n/transifex-pull.js

```diff
@@ -106,6 +107,9 @@ const getTranslationDownloadStatus = async (language, downloadRequestId) => {
     const languages = await getLanguages(organization, project);
     shell.echo('translations found:', languages.join(', '));
 
+    // Remove data managed on Transifex to avoid accumulation of vestigial files
+    shell.rm(path.join(translationsDirectory, '!(en).json'));
```

Comment:
```suggestion
    const translationFilenames = await fs.readdir(translationsDirectory);
    for (const filename of translationFilenames) {
        if (filename === 'en.json') {
            continue;
        }
        await fs.unlink(path.join(translationsDirectory, filename));
    }
```

Reply:
> How about this adjustment to your suggestion

Yes. Thanks for the correction. I did not know the readme was in the same folder.

Reply:
OK. I have updated the PR according to your suggestion:
https://github.com/arduino/arduino-ide/compare/a861cca617717c3b71f9080ea665a4b456af9eeb..d3c96148a457650ca350fd4aa7633bb0d17c594f

Thanks!